MetaMask/metamask-extension
# MetaMetrics: add segment.identify

## Explanation

Fixes https://github.com/MetaMask/metamask-extension/issues/13529

## More information

Support `segment.identify`, analytics-node method in our MetaMetrics controller

## Manual testing steps

1. Ensure Settings > Security & Privacy > "Participate in MetaMetrics" is turned on
2. Update one of the four settings:
  - metamaskState.ledgerTransportType,
  - number of accounts
  - network chains
  - metamaskState.threeBoxSyncingAllowed
3. Go to MixPanel and ensure related user has the updated settings
